commit aee0e0cedad91278299550de0de109ec15a789f7
Author: Adriaan de Groot <groot@kde.org>
Date:   Mon Dec 6 16:08:34 2021 +0100

    Fix build with -Werror
    
    - Although this function isn't called, building praat with
      picky flags -- such as -Werror -- fails because the function
      doesn't return anything.
diff --git LPC/PowerCepstrum.cpp LPC/PowerCepstrum.cpp
index a00890c28..6358ecca3 100644
--- LPC/PowerCepstrum.cpp
+++ LPC/PowerCepstrum.cpp
@@ -397,7 +397,7 @@ static autoMAT PowerCepstrum_getRhamonicsPower (PowerCepstrum me, double pitchFl
 }
 
 double PowerCepstrum_nearestPeak (PowerCepstrum me, double quefrency) {
-	
+	return 0.0;
 }
 
 autoTable PowerCepstrum_to_Table_rhamonics (PowerCepstrum me, double pitchFloor, double pitchCeiling, double f0fractionalWidth) {
